installer/datafiles/base_omsagent.data

```diff
@@ -180,6 +180,20 @@ fi
 # The real conf directory has to be owned by omsagent to be readable by the omsagent daemon
 chown -R omsagent:omsagent ${{CONF_DIR}}
 
+# Should run before onboarding script(Postinstall_300) and enabling OMS service(%Postinstall_500)
+%Postinstall_290
+if [ ${{OMSHOME}}/ruby/bin/gem list | grep -q "mysql2" ]; then
+    echo "MySQL ruby gem already installed...Skipping..."
+else
+	echo "----- Installing MySQL ruby gem -----"
+    if [! ${{OMSHOME}}/ruby/bin/gem install mysql2 2>/dev/null ]; then
```

Comment:
Inconsistent indentation, please uses spaces for indentation. You're using tabs for some of this, causing the problem.